### PR TITLE
Add "Used by" section on landing page

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Changes proposed in this pull request:
- Add `Used by` section on landing page,
- Create appropriate folder in `content` dir for aggregating adapters,
- Update appropriate json in `content/i18n` folder.

Instruction about how to add new case study is [here](https://github.com/kyma-project/website/pull/454).

**Related issue(s)**
Part of https://github.com/kyma-project/website/issues/363

### CODE REVIEW

File: src/views/landingPage/index.tsx [hunk 1]
```diff
@@ -9,21 +9,23 @@ import { MakeSpecial } from "./MakeSpecial";
...
   PageContext<LandingPageContext>
-> = ({ pageContext: { earlyAdopters } }) => (
+> = ({ pageContext: { earlyAdopters, adopters } }) => (
   <>
     <Headline />
     <WhatIs />
     <MakeSpecial />
     <KeyFeatures />
-    <CommunityAndEarlyAdoptersWrapper>
+    {/* <CommunityAndEarlyAdoptersWrapper>
```
Comment (aerfio) [inline src/views/landingPage/index.tsx]: remove that comment

Comment (magicmatatjahu) [reply src/views/landingPage/index.tsx]: `Used by` section will be visible after your changes, so on PR old section community and old adopters  are visible, but before merging I'll comment new adopters section and uncomment this code :)

File: src/views/landingPage/Adopters/index.tsx [hunk 2]
```diff
@@ -35,6 +37,15 @@ export const Adopters: React.FunctionComponent<Props> = ({ adopters }) => (
         <Grid.Unit df={12}>
           <AdoptersGallery adopters={adopters} />
         </Grid.Unit>
+        <Grid.Unit df={12}>
+          <AddYourCompanyButton>
+            <Link.External to="">
```
Comment (aerfio) [inline src/views/landingPage/Adopters/index.tsx]: update this link

File: src/views/landingPage/Adopters/styled.ts [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+
+export const StyledAdoptersItemExtLink = styled(Link.External)`
+  ${linkEffect}
+`;
+
+export const StyledAdoptersItemIntLink = styled(Link.Internal)`
+  ${linkEffect}
+`;
+
+export const AddYourCompanyButton = styled.div`
```
Comment (aerfio) [inline src/views/landingPage/Adopters/styled.ts]: ...Wrapper

File: src/views/landingPage/index.tsx (hunk 1 above)
Comment (aerfio) [reply src/views/landingPage/index.tsx]: as with the lastest agreement please just remove everything that is commented out, I mean delete those files because they are not longer needed

File: gatsby/constants.ts [hunk 4]
```diff
@@ -19,14 +20,17 @@ const DOCS_COMPONENTS_TYPE = "components";
 const DOCS_KYMA_ID = "kyma";
 const DOCS_SPECIFICATIONS_PATH = "specifications";
 
+const ADOPTER_FILENAME_REGEX = /adopters\/(.*?)\/index\.md$/;
```
Comment (aerfio) [inline gatsby/constants.ts]: .*? also accepts white space, should it be allowed?

File: gatsby/node-api/pages/landingPage/helpers.ts [hunk 5]
```diff
@@ -33,3 +40,38 @@ function getEarlyAdopters(): EarlyAdopter[] {
   const data = safeLoad(file) as { adopters: EarlyAdopter[] };
   return JSON.parse(JSON.stringify(data.adopters));
 }
+
+export async function getAdopters(
+  graphql: GraphQLFunction,
```
Comment (aerfio) [inline gatsby/node-api/pages/landingPage/helpers.ts]: why did you add graphql as argument if you are not using it?

File: src/views/landingPage/Adopters/AdoptersGallery.tsx [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+  return (
+    <AdoptersGalleryWrapper>
+      <Grid.Container>
+        <Grid.Row>
+          <Grid.Unit
+            df={1}
+            xs={0}
+            className="carousel-buttons-nav carousel-button-left"
+          >
+            <div>
```
Comment (aerfio) [inline src/views/landingPage/Adopters/AdoptersGallery.tsx]: empty div?

Comment (magicmatatjahu) [reply src/views/landingPage/Adopters/AdoptersGallery.tsx]: Yes, I need wrap Button with default block element to remove flex properties.

**Comment by magicmatatjahu** [comment]
@aerfio 
```
Also, is it intentional that when you click right arrow the carousel slides to the left?
```
Yes
